src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -98,7 +100,23 @@ public void ClientAsyncAuthenticate_MismatchProtocols_Fails(SslProtocols server,
             Assert.Throws(expected, () => ClientAsyncSslHelper(server, client));
         }
 
+        [Theory]
+        [MemberData("ProtocolMismatchDataSsl2Specific")]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
This is Windows-specific because we don't expect these to fail on Linux?  Why is that?